2-Authorization/3-call-own-api-dotnet-core-daemon/README.md

```diff
@@ -171,6 +171,8 @@ Open the project in your IDE (like Visual Studio or Visual Studio Code) to confi
 1. Find the placeholder `Enter_the_Tenant_Id_Here` and replace the existing value with your Azure AD tenant/directory ID.
 1. Find the placeholder `Enter_the_Tenant_Subdomain_Here` and replace it with the Directory (tenant) subdomain. For instance, if your tenant primary domain is *contoso.onmicrosoft.com*, use *contoso*.
 
+    > To use a CIAM custom domain instead, you can replace the entire `https://Enter_the_Tenant_Subdomain_Here.ciamlogin.com/` placeholder with the (custom domain)/tenant/v2.0. For instance, if you custom domain is *contoso.ciamcustom.com*, you can use `https://contoso.consotoextensibility.com/tenant/v2.0`.
```

Comment:
I guess this doesn't work so skip this update and open a bug. We can prioritize as needed.

Reply:
ok